Packs/OpenPhish/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### New: OpenPhish v2
+- OpenPhish uses proprietary Artificial Intelligence algorithms to automatically identify zero-day phishing sites and provide comprehensive, actionable, real-time threat intelligence.
```

Comment:
```suggestion
- Added a v2 for the OpenPhish integration that provides enrichment for URLs.
```

Reply:
@kirbles19 your inputs

Reply:
Created v2 of the OpenPhish integration, which now provides URL enrichment.